style.css

```
@@ -0,0 +1,27 @@
+h1{
+    color:blue
+}
```

Always end your rules with a semicolon:
```suggestion
h1{
    color:blue;
}
```